libs/sdk-ui-dashboard/src/presentation/scheduledEmail/DefaultScheduledEmailDialog/ScheduledMailDialogRenderer/ScheduledMailDialogRenderer.tsx

```diff
@@ -597,22 +614,19 @@ export class ScheduledMailDialogRendererUI extends React.PureComponent<
     };
 
     private onDateChange = (selectedDateObject: Date): void => {
-        this.setState((prevState) => {
-            return {
-                repeat: {
-                    ...prevState.repeat,
-                    date: {
```

Comment:
found by accident when comparing states - the `date` property is not in the type and is newer used

Reply:
moved whole `updateStartDateForRepeats` to this function